[2022-09-05 14:34:46,637] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_tables.test scheduled__2022-09-03T00:00:00+00:00 [queued]>
[2022-09-05 14:34:46,927] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_tables.test scheduled__2022-09-03T00:00:00+00:00 [queued]>
[2022-09-05 14:34:46,944] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 14:34:46,960] {taskinstance.py:1377} INFO - Starting attempt 7 of 7
[2022-09-05 14:34:46,976] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 14:34:47,434] {taskinstance.py:1397} INFO - Executing <Task(MySqlOperator): test> on 2022-09-03 00:00:00+00:00
[2022-09-05 14:34:48,787] {standard_task_runner.py:52} INFO - Started process 1244 to run task
[2022-09-05 14:34:48,852] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'create_tables', 'test', 'scheduled__2022-09-03T00:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/create.py', '--cfg-path', '/tmp/tmpyeucgccn', '--error-file', '/tmp/tmpuviavzji']
[2022-09-05 14:34:48,865] {standard_task_runner.py:80} INFO - Job 137: Subtask test
[2022-09-05 14:35:02,686] {task_command.py:371} INFO - Running <TaskInstance: create_tables.test scheduled__2022-09-03T00:00:00+00:00 [running]> on host cdeef4137858
[2022-09-05 14:35:04,538] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_tables
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-09-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=7
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-03T00:00:00+00:00
[2022-09-05 14:35:04,539] {mysql.py:82} INFO - Executing: SHOW VARIABLES LIKE "secure_file_priv";
[2022-09-05 14:35:04,719] {base.py:68} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-09-05 14:35:05,442] {dbapi.py:231} INFO - Running statement: SHOW VARIABLES LIKE "secure_file_priv";, parameters: None
[2022-09-05 14:35:05,645] {dbapi.py:239} INFO - Rows affected: 1
[2022-09-05 14:35:05,734] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=create_tables, task_id=test, execution_date=20220903T000000, start_date=20220905T143446, end_date=20220905T143505
[2022-09-05 14:35:06,029] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05 14:35:06,270] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
